bootstrap/bootstrap.go

```diff
@@ -219,16 +199,39 @@ func (b *Bootstrap) StartAPIServer(ctx context.Context) error {
 		nil,
 	)
 
+	accountKeys := make([]*requester.AccountKey, 0)
+	account, err := b.client.GetAccount(ctx, b.config.COAAddress)
+	if err != nil {
+		return fmt.Errorf(
+			"failed to get signer info account for address: %s, with: %w",
+			b.config.COAAddress,
+			err,
+		)
+	}
+	signer, err := createSigner(ctx, b.config, b.logger)
+	if err != nil {
+		return err
+	}
+	for _, key := range account.Keys {
```

Comment:
**Note:** Maybe we should leave out the first couple keys (e.g. 5), so that the Flow account can also sign transactions outside the scope of the EVM Gateway, if the account owner has to do so.